components/sup/protocols/event.proto

```diff
@@ -98,4 +98,6 @@ message HealthCheckEvent {
   google.protobuf.StringValue stdout = 6;
   // The health check hook stderr output
   google.protobuf.StringValue stderr = 7;
+  // The heath check interval in seconds
+  uint64 interval = 8;
```

Comment:
This might be better as a `google.protobuf.Duration`, since that removes any ambiguity around the units (we also use this above for the `execution` field).

Reply:
Good point - will get that in there :)